app/views/donations/index.html.erb

```diff
@@ -1,13 +1,52 @@
-<div align="center">
-<div class="container-fluid" style="padding-top: 20px; padding-left:; width: 30%; float: centered;">
-	<h1 class="container-fluid"> Support Our Cause </h1>
+
+
+<div align="center" style= "padding-top: 20px; border: solid; background: #129EA2; color: #129EA2; border-width: .5px;" >
+	<div class="top-right"  >
+		<div class="container-fluid" style="margin-top: 3%; padding-bottom: 45px; width: 30%; float: right; margin-right: 20%">
+			<%= image_tag("/wicilogo.png", :alt => "rss feed", align: "center", style: "float: centered; margin-bottom: 10px; width: 70%; height 70%;") %>
+			<div style= "float:left; font-size: 12px; ">
+				The Women in Chemistry Initiative is a non-profit 501c3 organization.  All donations are tax deductible.  For more information on tax deductions for 501c3 non-profits, please visit the following website.
+			</div>
+			<%= link_to "https://501c3go.com/what-is-a-501c3/", style: "font-size: 12px" %>
+	</div>
+</div>
+</div>
+
+
+<div class="container-fluid" style="padding-top: 20px; margin-top: 5%; background: #ffffff; padding-bottom: 10px; width: 30%; float: left; border: solid; border-width: 1px; border-color: #D3D3D3; border-radius: 5px; margin-left: 20%; text-align: center;">
+	<h1 class="container-fluid" style="padding-bottom: 1%; font-size: 25px; border-bottom: solid; border-bottom-width: 1px; border-bottom-color: #D3D3D3"> Support Our Cause </h1>
 	<div class="container-fluid"> Proceeds go towards the Women in Chemistry Initiative </div>
 
 	<%= form_tag(new_donation_path, method: :get, class: "form-group") do %>
-	  <%= text_field_tag :amount_dollars, 0, class: "form-control", style: "margin-top: 10px;"%>
-	  <%= submit_tag "Donate", id: :donate, class: "btn btn-primary form-control", style: "margin-top: 10px;"%>
+	<div class="input-group mb-3" style= "display: inline-block;">
+  		<div class="input-group-prepend">
+  			<span style= "margin-right: -10px; position: absolute; padding-left: 1%; padding-top: 4.7%;">$</span>
+  			<%= text_field_tag :amount_dollars, 0, class: "donatetext", style: "border-radius: 5px; padding-left: 20px;"%>
```

Comment:
Dollar sign is misaligned in here 
![image](https://user-images.githubusercontent.com/20048451/57579083-d7004880-744b-11e9-84b6-d32f47eacdbc.png)